### ISSUE TITLE
Yolo - Zed Camera

### ISSUE THREAD

**Issue by lucCult** [pjreddie/darknet#522]
Hi, I'm trying to use Zed Camera with YOLO, how I can use Zed Camera API (that are written in C#) in demo.c ?

**Comment by TheMikeyR** [comment]
I thought the API is written in cpp with wrappers for ROS, OpenCV etc.?

**Comment by lucCult** [comment]
Yes, the API is written in cpp with wrappers for ROS, Python, Unity. But I want to use the API code in YOLO  (in particular demo.c class) written in C.

**Comment by ouyangkid** [comment]
I use the python to connect with my avt camera through the ROS. I am not so strong to handle pure C with opencv.

**Comment by lucCult** [comment]
I'm trying to find a way to use zed API in YOLO source code.

**Comment by MizbaMohammed** [comment]
Hi everyone, is anyone able to figure out how to use Zed camera API in demo.c?
